test/test.js

```diff
@@ -162,13 +149,15 @@ try {
     if (sentry.length) logOnDetails.sha_sentryfile = sentry;
 }
 catch (beef){
-    util.log("Cannae load the sentry. " + beef);
+    util.log("Cannot load the sentry. " + beef);
```

Comment:
It's already fubarred sync wise. I'll just make a note to fix it up when I have to do the re-base and merge.